hubble-proxy/cmd/version/version.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package version
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/cilium/cilium/pkg/version"
+
+	"github.com/spf13/cobra"
+)
+
+// New creates a new version command.
+func New() *cobra.Command {
+	return &cobra.Command{
+		Use:   "version",
+		Short: "Display detailed version information",
+		Long:  `Displays information about the version of this software.`,
+		Run: func(_ *cobra.Command, _ []string) {
+			runVersion(os.Stdout)
```

Comment:
there is an stdout attached to the cmd object, rather than os (useful to get into the habbit of that for future testing implications)